pkg/interactive/cobra.go

```diff
@@ -0,0 +1,63 @@
+package interactive
+
+import (
+	"fmt"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+func AskMissingRequiredFlags(command *cobra.Command) {
+	if getSettings().NonInteractive {
+		return
+	}
+
+	command.Flags().VisitAll(flagOrInteractive)
+}
+
+func flagOrInteractive(flag *pflag.Flag) {
+	if flag.Changed {
+		return
+	}
+
+	required, hasRequired := flag.Annotations[cobra.BashCompOneRequiredFlag]
+	if !hasRequired {
+		return
+	}
+
+	if required[0] != "true" {
```

Comment:
`util.trueStr`?

Reply:
Added constants for both "true" and "false" 👍